mdal/mdal.cpp

```diff
@@ -274,27 +278,27 @@ void MDAL_M_LoadDatasets( MeshH mesh, const char *datasetFile )
 {
   if ( !datasetFile )
   {
-    sLastStatus = MDAL_Status::Err_FileNotFound;
+    MDAL::Log::error( MDAL_Status::Err_FileNotFound, "Dataset file is not valid (null)" );
     return;
   }
 
   if ( !mesh )
   {
-    sLastStatus = MDAL_Status::Err_IncompatibleMesh;
+    MDAL::Log::error( MDAL_Status::Err_IncompatibleMesh, "Mesh is not valid (null)" );
     return;
   }
 
   MDAL::Mesh *m = static_cast< MDAL::Mesh * >( mesh );
 
   std::string filename( datasetFile );
-  MDAL::DriverManager::instance().loadDatasets( m, datasetFile, &sLastStatus );
+  MDAL::DriverManager::instance().loadDatasets( m, datasetFile, &MDAL::Log::sLastStatus );
```

Comment:
MDAL::Log::sLastStatus should be hidden from all world and used only in the mdal_logger.cpp file . Status can be set with MDAL::Log:error in the loadDatasets() similar to the loadMesh

Reply:
yep, it is erased in new commit